composer.lock

```diff
@@ -13608,5 +13608,5 @@
     "platform-overrides": {
         "php": "7.2.5"
     },
-    "plugin-api-version": "2.6.0"
```

Comment:
I'm not so sure about the `plugin-api-version` downgrade, could this be linked to my composer version ? (2.5.8)

Reply:
@matthieu-rolland Yep you must update your composer to current version, then it will be 2.6.0. Or just manually edit the line in `composer.lock`.

Reply:
yep, updated composer and it's all good 👍